src/main/scala/is/hail/io/vcf/ExportVCF.scala

```diff
@@ -44,9 +44,13 @@ object ExportVCF {
       case TString(_) =>
         sb.append(TString.loadString(m, offset))
       case TCall(_) =>
-        val p = Genotype.gtPair(m.loadInt(offset))
+        val c = m.loadInt(offset)
+        val p = Call.gtPair(c)
```

Comment:
gtPair => allelePair?  I feel like the letters g and t should never appear next to each other.